tests/integration/test_schema_compliance.py

```
@@ -0,0 +1,483 @@
+    def visit_FunctionDef(self, node):
+        """Handle default values in function arguments."""
+        # Check defaults
+        for default in node.args.defaults:
+            self._check_value(default, getattr(node, "lineno", 0))
+        # Check kw_defaults
+        for default in node.args.kw_defaults:
+            if default:
+                self._check_value(default, getattr(node, "lineno", 0))
+        self.generic_visit(node)
+
+    def _check_value(self, node, lineno):
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `_check_value`

Add return type annotation: `None`

(ANN202)